lib/dialyzer/src/dialyzer_typegraph.erl

```diff
@@ -29,7 +29,7 @@ module_type_deps(UseContracts, CodeServer, Modules) ->
   Contracts =
     case UseContracts of
       true -> maps:from_list(dict:to_list(dialyzer_codeserver:get_contracts(CodeServer)));
-      false -> []
+      false -> #{}
```

Comment:
What impact did this bug have in practice?

Reply:
If the `--no_spec` option was given, this bug caused Dialyzer to crash in a subsequent call to `maps:to_list/1`.